metagraph/src/graph/representation/hash/dbg_sshash.cpp

```diff
@@ -99,6 +100,35 @@ void DBGSSHash::add_sequence(std::string_view sequence,
     throw std::logic_error("adding sequences not supported");
 }
 
+template <class Dict, bool with_rc>
+void map_to_nodes_with_rc_advanced(size_t k,
```

Comment:
Possibly rename to `map_to_nodes_with_rc_internal` or `map_to_nodes_with_rc_impl`.

Reply:
Ok, let's do `impl`. I think it better conveys our intention for why this function exists.